canonical-web-and-design/vanilla-framework
# Add side navigation updates to component status page

## Done

Leftover of #3052 
Changes to side navigation are now listed on component status page


## QA

- Run `./run` or [demo](https://vanilla-framework-canonical-web-and-design-pr-3064.run.demo.haus/docs/component-status)
- Check the [component status page](https://vanilla-framework-canonical-web-and-design-pr-3064.run.demo.haus/docs/component-status) - make sure latest changes to side navigation are listed
